congestion-control-charter.txt

```diff
@@ -1,85 +1,32 @@
-
-* Techniques to address lower-layer alterations to packet timing or ordering.
-
-* New ways for endpoints to respond to both implicit and explicit congestion
-signals, including specification of explicit signals.
-
-* Progression of existing Informational or Experimental RFCs to higher maturity,
-if they meet the criteria.
-
-* Real-time, media adaptation algorithms for peer to peer communications
-
-* Operational guidance, in general, including advice to upper layers or new
-transport protocols. Examples include minimum frequency of feedback,
-counterproductive application retransmissions and multiple connections to the
-same host.
-
-Proposals that depend on the capabilities of a single transport protocol should
-generally remain in the maintenance working group for that protocol (i.e.,
-TCPM, QUIC, TSVWG). Documents in CONGRESS should remain as independent of
-transport protocol as possible, but they might have short specific instructions
-(e.g. a header option or parameter format) for one or more protocols.
-
-CONGRESS is not chartered to publish Informational RFCs documenting the state of
-congestion control in the Internet, including assessments of compliance with
-existing standards. Other venues, e.g., IRTF, may be more appropriate for
-publishing such documents.
-
-CONGRESS will not remain open simply “in case” further work comes along. 
+﻿Congestion Working Group Charter
+
+RFC 5033 describes a Best Current Practice to evaluate new congestion control
+algorithms as Experimental or Proposed Standard RFCs. TCP was the dominant
+consumer of this work, and proposals were typically discussed in research
+groups (for example, Internet Congestion Control Research Group - ICCRG).
+
+Since RFC 5033 was published, many conditions have changed. Congestion control
+algorithm proponents now often have the opportunity to test and deploy at scale
+without IETF review. The set of protocols using these algorithms has spread
+beyond TCP and SCTP to include DCCP, QUIC, and beyond. There is more interest
+in specialized use cases such as data centers and real-time protocols. Finally,
+the community has gained much more experience with indications of congestion
+beyond packet loss.
+
+The Congestion working group will review some of the impediments to congestion
+control work occurring in the IETF, and can generalize transport in this area
+from TCP to all of the relevant transport protocols. This will inform a revision
+of RFC 5033 that encourages IETF review of congestion control proposals and
+standardization of mature congestion control algorithms.
+
+The congestion control expertise in the working group would also make it a
+natural venue to take on other work related to indications of congestion (such
+as delay), queueing algorithms, rate pacing, interaction with other layers,
+etc. In particular, it could address congestion control algorithms mature enough
+for standardization. The working group is intended to be a home for such work,

```

Comment:
I do think it's worth to clarify the "mature enough for standardization" with the two criteria we had in the old version, just to make it clearer for everyone what the distinction is between this WG and ICCRG/IRTF work: 
```
(1) empirical evidence of safety (for example - avoiding congestion collapse) and 
(2) stated intent to deploy by major transport implementations.
```

Related to Éric's comment, I wonder if we should mention experimentation as an example of empirical evidence, but not the only one -- I'd say having an algorithm deployed in production for a certain period of time can also serve as empirical evidence, even in the absence of a published formal experimentation methodology. (That's why I don't want to simply change "empirical" to "experimental".)